src/jmcomic/jm_plugin.py

```
@@ -1216,3 +1216,28 @@ def new_decide_dir(photo, ensure_exists=True) -> str:
             return original_path
 
         self.option.decide_image_save_dir = new_decide_dir
+
+
+class AdvancedRetryPlugin(JmOptionPlugin):
+    plugin_key = 'advanced-retry'
+
+    def invoke(self,
+               retry_config,
```

Ruff (0.12.2)

Unused method argument: `retry_config`

(ARG002)